assets/in

```diff
@@ -63,8 +63,8 @@ if [ -n "$branch" ]; then
   branchflag="--branch $branch"
 fi
 
-depthflag=""
-if test "$depth" -gt 0 2> /dev/null; then
+depthflag="--depth $depth"
+if test "$depth" -ne 1 2> /dev/null; then
   depthflag="--depth $depth"
 fi
```

Comment:
Since both branches of the iff statement do the same thing, it can be dropped completely!
```suggestion
```